src/PIL/PSDraw.py

```diff
@@ -28,15 +28,12 @@
 class PSDraw:
     """
     Sets up printing to the given file. If ``fp`` is omitted,
-    ``sys.stdout.buffer`` or ``sys.stdout`` is assumed.
+    ``sys.stdout.buffer`` is assumed.
     """
 
-    def __init__(self, fp=None):
+    def __init__(self, fp: IO[bytes] | None = None) -> None:
         if not fp:
-            try:
-                fp = sys.stdout.buffer
-            except AttributeError:
-                fp = sys.stdout
```

Comment:
When mypy looks at `sys.stdout` here, it says
> expression has type "Union[TextIO, Any]"

I think #5437 misinterpreted the docs, and that really, `sys.stdout` will not accept the bytes that we are sending.

https://docs.python.org/3/library/sys.html#sys.stdout
> be aware that the standard streams may be replaced with file-like objects like [io.StringIO](https://docs.python.org/3/library/io.html#io.StringIO) which do not support the buffer attribute.